lib/hanami/cli/application/commands/console.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "hanami/cli"
+require "hanami/cli/application/command"
+require "hanami/console/context"
+
+module Hanami
+  class CLI
+    module Application
+      module Commands
+        class Console < Command
+          REPL = begin
+            require 'pry'
+            Pry
+          rescue LoadError
+            require 'irb'
+            IRB
+          end
+
+          desc "Open interactive console"
+
+          def call(**)
+            measure "#{prompt_prefix} booted in" do
+              out.puts "=> starting #{prompt_prefix} console"
+              application.boot
+            end
```

Comment:
At work, I built an app console (also using Hanami::CLI) that does not boot the container and I simply rely on lazy-booting of everything. It starts under 1s and I didn't have any issues with it. I'd encourage you to try the same approach because people will love the fast booting aspect of it.

Reply:
Oh yes! Thank you for the reminder about this. I've been meaning to do it with my work apps, and in the meantime I'll just go ahead and make the change directly here.

Reply:
@solnic crazy idea: I wonder if we could rely on zeitwerk additionally in the console (and _only_ in the console, mind you) to auto-load any missing constants. That way you have the benefit of lazy loading for resolving container registrations _and_ any direct class lookups...

Reply:
@timriley hmm yeah, this could be useful for global constants like struct classes or schemas, so I guess it's at least worth a shot. I don't think it will improve performance in any significant way though. The only benefit is going to be that you won't have to manually require some files, which is definitely good UX in a console.

Reply:
@jodosha I'll leave it like it is for now, and we can have a future discussion about benefits of the approach @solnic mentioned. This will mean the difference of the console being ready in 1 second versus 5+ seconds for larger applications, which is a big deal IMO.